pandas/core/reshape/concat.py

```diff
@@ -14,9 +14,12 @@
     cast,
     overload,
 )
+import weakref
 
 import numpy as np
 
+from pandas._config import get_option
```

Comment:
```suggestion
from pandas._config import get_option, using_copy_on_write
```

Reply:
Ah this is annoying, I thought that ci should fail, but makes sense that the import continues to work through there...